modules/utxo-lib/src/bitgo/wallet/chains.ts

```diff
@@ -110,3 +110,11 @@ export function isSegwit(v: ChainCode): boolean {
   const segwitCodes: ChainCode[] = [...chainCodesP2shP2wsh, ...chainCodesP2wsh, ...chainCodesP2tr];
   return segwitCodes.includes(v);
 }
+
+/**
+ * Prepend bip32 derivation path with 'm/' if it is not there
+ * @param path
+ */
+export function formatCorrectBip32DerivationPath(path: string): string {
+  return path.slice(0, 2) !== 'm/' ? 'm/' + path : path;
+}
```

Comment:
as discussed on slack - paths that don't start with `m/` must be supported, just adding `m/` is not a solution here

Reply:
Looking upstream now